back/engines/commercial/project_management/spec/policies/phase_policy_spec.rb

```diff
@@ -18,6 +18,7 @@
       it { is_expected.to     permit(:create)  }
       it { is_expected.to     permit(:update)  }
       it { is_expected.to     permit(:destroy) }
+      it { is_expected.to     permit(:delete_inputs) }
```

Comment:
Isn't this file incomplete? It does not describe the behaviour for non-moderators.

Reply:
Which case is missing? There are visitors and different kinds of mortals (I wanted to rename this one, but forgot what's now our official name for normal citizens?).  There is also one for project moderators. Do you mean that one for folder moderators is missing and perhaps for moderators of a different project/folder?

Reply:
"Residents"

Reply:
I meant that the file only contains `context 'for a moderator'`, not for residents, who are not allowed to delete inputs.

Reply:
We discussed it and decided that the current test suite is enough.